lib/main.coffee

```diff
@@ -43,6 +43,7 @@ module.exports =
     helpers = require('atom-linter')
     path = require('path')
     provider =
+      name: 'shellcheck'
```

Comment:
It looks like the name should actually be `ShellCheck` from their repo here: https://github.com/koalaman/shellcheck